client/client-multi/client-objective-c/Kaa/channel/failover/DefaultFailoverManager.m

```diff
@@ -166,129 +157,97 @@ - (void)onServerChangedWithConnectionInfo:(id<TransportConnectionInfo>)connectio
+                DDLogVerbose(@"%@ Cancelling fail resolution: %@", logTag, pointResolution);
                 [self cancelCurrentFailResolution:pointResolution];
             }
             AccessPointIdResolution *newPointResolution =
             [[AccessPointIdResolution alloc] initWithAccessId:[connectionInfo accessPointId] resolution:nil];
             self.resolutionProgressMap[serverTypeKey] = newPointResolution;
         } else {
-            DDLogDebug(@"%@ Same server [%@] is used, nothing has changed", TAG, connectionInfo);
+            DDLogDebug(@"%@ Same server [%@] is used, nothing has changed", logTag, connectionInfo);
         }
     }
 }
 
 - (void)onServerConnectedWithConnectionInfo:(id<TransportConnectionInfo>)connectionInfo {
     
-    DDLogVerbose(@"%@ Server %@ has connected", TAG, connectionInfo);
+    DDLogVerbose(@"%@ Server %@ has connected", logTag, connectionInfo);
     if (!connectionInfo) {
-        DDLogWarn(@"%@ Server connection info is nil, can't resolve", TAG);
+        DDLogWarn(@"%@ Server connection info is nil, can't resolve", logTag);
         return;
     }
     
+    [self.failoverStrategy onRecoverWithConnectionInfo:connectionInfo];
+    
     @synchronized(self) {
         AccessPointIdResolution *pointResolution = self.resolutionProgressMap[@([connectionInfo serverType])];
         if (!pointResolution) {
             DDLogVerbose(@"%@ Server hasn't been set (failover resolution has happened), new server %@ can't be connected",
-                         TAG, connectionInfo);
+                         logTag, connectionInfo);
         } else if (pointResolution.resolution && pointResolution.accessPointId == [connectionInfo accessPointId]) {
-            DDLogVerbose(@"%@ Cancelling fail resolution: %@", TAG, pointResolution);
+            DDLogVerbose(@"%@ Cancelling fail resolution: %@", logTag, pointResolution);
             [self cancelCurrentFailResolution:pointResolution];
         } else if (pointResolution.resolution) {
             DDLogDebug(@"%@ Connection for outdated accessPointId: %i was received - ignoring. New accessPointId: %i",
-                       TAG, [connectionInfo accessPointId], pointResolution.accessPointId);
+                       logTag, [connectionInfo accessPointId], pointResolution.accessPointId);
         } else {
             DDLogVerbose(@"%@ There is no current resolution in progress, connected to the same server: %@",
-                         TAG, connectionInfo);
+                         logTag, connectionInfo);
         }
     }
 }
 
 - (FailoverDecision *)decisionOnFailoverStatus:(FailoverStatus)status {
     
-    DDLogInfo(@"%@ Applying failover strategy for status: %i", TAG, status);
+    DDLogVerbose(@"%@ Applying failover decision for status: %i", logTag, status);
 
     @synchronized(self) {
-        NSNumber *serverTypeKey = nil;
+        AccessPointIdResolution *accessPointIdResolution = nil;
```

Comment:
Can't see where this is variable is used. It is created and modified, but is it passed anywhere?